### PR TITLE
Add Deductive Reasoning Scenario and One Related Subtask

### PR DESCRIPTION
**Pull request by Jianqiao-Zhao**

### CODE REVIEW

File: src/helm/benchmark/scenarios/cleva_scenario.py [hunk 1]
```diff
@@ -918,3 +918,24 @@ class CLEVABiasScenario(CLEVAScenario):
...
+
+        问题：根据陈述1.和2.，结论是否正确？
+        A. 否
+        B. 是
+
+    Target: B
+    """
+
+    description = "Deductive reasoning task in CLEVA benchmark"
+    tags = ["deductive_reasoning", "multiple_choice"]
```
Comment (lyy1994) [inline src/helm/benchmark/scenarios/cleva_scenario.py]: You can add a "reasoning" tag here.

File: src/helm/benchmark/scenarios/cleva_scenario.py [hunk 2]
```diff
@@ -918,3 +918,24 @@ class CLEVABiasScenario(CLEVAScenario):
 
     description = "Bias task in CLEVA benchmark"
     tags = ["bias", "harms", "multiple_choice"]
+
+
+class CLEVADeductiveReasoningScenario(CLEVAScenario):
+    """
+    The deductive reasoning task of CLEVA benchmark.
+
+    An example is:
```
Comment (lyy1994) [inline src/helm/benchmark/scenarios/cleva_scenario.py]: Perhaps you can mark that this is a "modus_tollens " subtask example.

File: src/helm/benchmark/scenarios/cleva_scenario.py (hunk 1 above)
Comment (Jianqiao-Zhao) [reply src/helm/benchmark/scenarios/cleva_scenario.py]: Added in the latest version.

File: src/helm/benchmark/scenarios/cleva_scenario.py (hunk 2 above)
Comment (Jianqiao-Zhao) [reply src/helm/benchmark/scenarios/cleva_scenario.py]: Adjusted in the latest version.

**Review by lyy1994 — APPROVED** [review]

**Pull request merged**